src/profile-logic/process-profile.js

```diff
@@ -1040,6 +1040,7 @@ export function processProfile(
     logicalCPUs: geckoProfile.meta.logicalCPUs,
     // Gecko always sends the profile with URLs.
     networkURLsRemoved: false,
+    symbolicated: !!geckoProfile.meta.symbolicated,
```

Comment:
So actually I implemented this because some of the importers (most notably the linux perf format importer) output a profile in the gecko format, and for these ones I don't want to kick off the symbolication process. See https://github.com/firefox-devtools/profiler/pull/1522/files#diff-51a14db513e24ab9daf40718a7613fb8R236
All gecko profiles will have this property missing, which should then start the process.

Reply:
Because the 2  `symbolicated` booleans have different meanings, I'll change the name of the gecko profile's property to `presymbolicated`.